app/models/foreman_openscap/policy.rb

```diff
@@ -21,7 +21,7 @@ class Policy < ActiveRecord::Base
     SERVER_CLASS_PARAMETER   = 'server'
     PORT_CLASS_PARAMETER     = 'port'
 
-    validates :name, :presence => true, :uniqueness => true, :format => { :without => /\s/ }
+    validates :name, :presence => true, :uniqueness => true
```

Comment:
is there a reason why this existed in the first place? any limitation on scap scanner itself?